### PR TITLE
feat: upgrade federated assets source (AR-1563)

### PR DESCRIPTION
**Pull request by yamilmedina**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1563" title="AR-1563" target="_blank">StoryAR-1563</a>  I want to be able to send and receive assets to/from other backends
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Modifications required since some core lib module changed, namely: AssetsId Plain to Qualified.
Enabling interop in federated environments

### Dependencies (Optional)

https://github.com/wireapp/kalium/pull/578/files

Needs releases with: https://github.com/wireapp/kalium/pull/578/files

- [x] GitHub link to other pull request

#### How to Test

You should, Login in a federated environment and be able to send and receive assets from external endpoints

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-653/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3590699)(3590699)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3590699)(3590699)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-653/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3590795)(3590795)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3590795)(3590795)-dev-release.apk)

File: app/src/test/kotlin/com/wire/android/mapper/MessageContentMapperTest.kt [hunk 1]
```diff
@@ -80,21 +83,21 @@ class MessageContentMapperTest {
...
-                    && resultContentLeft.author.asString(arrangement.resources) == member1.otherUser.name
+            resultContentLeft is ServerMessage.MemberLeft &&
+                resultContentLeft.author.asString(arrangement.resources) == member1.otherUser.name
         )
         assert(
-            resultContentRemoved is MessageViewContent.ServerMessage.MemberRemoved
-                    && resultContentRemoved.author.asString(arrangement.resources) == member1.otherUser.name
-                    && resultContentRemoved.memberNames.size == 1
-                    && resultContentRemoved.memberNames[0].asString(arrangement.resources) == member2.otherUser.name
+            resultContentRemoved is com.wire.android.ui.home.conversations.model.MessageContent.ServerMessage.MemberRemoved &&
```
Comment (gongracr) [inline app/src/test/kotlin/com/wire/android/mapper/MessageContentMapperTest.kt:90]: Maybe add this as import?

**Review by gongracr — APPROVED** [review]
Great business as usual 💼 🤵‍♂️

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-653/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3622872)(3622872)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3622872)(3622872)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-653/5/) **failed**.

**Comment by AndroidBob** [comment]
Build [9](https://10.10.124.134/job/wire-android-reloaded/job/PR-653/9/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3623271)(3623271)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3623271)(3623271)-dev-release.apk)

**Pull request merged**